sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/AdvancedDiffLabeledUnlabeledData.java

```diff
@@ -17,6 +18,14 @@
 /**
  * Sample to show the differences in output that arise when RecognizeCustomForms
  * is called with custom models trained with labeled and unlabeled data.
+ * For this sample, you can use the training forms found in https://aka.ms/azsdk/formrecognizer/docs/trainingdocs for
```

Comment:
Just FYI in case you didn't see... Mariana is going to check with Ted to see if we can get the aka.ms link working - the redirection seems to be making the training fail.

Reply:
yeah, looked at the other PR comments, will wait until this is figured out.

Reply:
We are good to go with the link. Ted fixed it :)

Reply:
I know.. sorry, I spoke too soon :(